tasks/34_max-triplet-sum/max-triplet-sum.js

```diff
@@ -0,0 +1,22 @@
+// c is variable for triplet = 3, quartet = 4, quintet = 5 sum
+function maxTripletSum(array, c = 3) {
+  if (array.length >= c) {
+    const newArray = array.sort(function(a, b) {
+      return b - a;
+    });
+    let tripletSum = 0;
+    let cycle = 0;
+    for (let i = 0; i < newArray.length; i++) {
```

Comment:
It could be simplified using ES6 features.

Reply:
really, after a few tasks, I understood how solve this task easier, thanks :)